### PR TITLE
Drop Leap-15.3 from OBS CI

### PR DESCRIPTION
**Pull request by mimi1vx**
Leap 15.3 is EOL for long time, no need test against it

### CODE REVIEW

**Comment by perlpunk** [comment]
I just wanted to suggest to also reduce the number here: https://github.com/os-autoinst/os-autoinst/blob/master/.mergify.yml#L17
But I saw that we now have 19 checks in this PR. I think that's more than normal because the branch was created in the original repo, and github reports any actions for both push and PR events. So that are 4 extra checks, normally we would have 15 then, that's still more than the 14 that we originally put into the mergify config. So I assume we have added checks after we added that to mergify. So maybe just increase the value to 15?

**Comment by perlpunk** [comment]
I just saw you reduced it to 13. Not sure why.

**Comment by mimi1vx** [comment]
> I just saw you reduced it to 13. Not sure why.

opposite action to time you added 15.4 : D

now so how successful checks are mandatory for mergify?

**Review by Martchus — APPROVED** [review]

File: .mergify.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ pull_request_rules:
         # "unresolvable" is not reported in general:
         # https://trello.com/c/0N3jHq5M/2257-report-back-to-scm-when-build-results-arent-failed-and-succeeded
         # So we need to require the number of tests explicitly:
-        - "#check-success>=14"
+        - "#check-success>=13"
```
Comment (perlpunk) [inline .mergify.yml:17]: I would try 15
```suggestion
        - "#check-success>=15"
```

**Review by perlpunk — APPROVED** [review]

**Comment by perlpunk** [comment]
This has to be manually merged because of the mergify config change, will do that

**Pull request merged**